app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,57 @@
+class Users::OmniauthCallbacksController < Devise::OmniauthCallbacksController
```

Comment:
This class seems to lack specs. Will it be possible to have specs for it, please?